hs_core/tests/api/native/test_add_resource_files.py

```diff
@@ -100,7 +100,7 @@ def test_add_files_over_quota(self):
 
         uquota = self.user.quotas.first()
         # make user's quota over hard limit 125%
-        uquota.user_zone_value = uquota.allocated_value * 1.3
```

Comment:
Missed an import https://github.com/hydroshare/hydroshare/blob/manual-irods-userzone-and-rm-federation/hs_core/tests/api/native/__init__.py#L19

Reply:
interesting flake8 didn't catch that. Nice eye!